docs/source/global_tracer.rst

```diff
@@ -1,12 +1,12 @@
 Global Tracer Object
 ====================
 
-Some features in VizTracer requires a ``VizTracer`` object so it's helpful to make the tracer object accessible globally.
+Some features in VizTracer require a ``VizTracer`` object so it's helpful to make the tracer object accessible globally.
 
 When you are using command line entry ``viztracer your_script.py``, you don't need to worry about it. The tracer will be
 automatically registered and you can access it from any file. 
 
-When you are using inline script, after you instantiate the object like ``tracer = VizTracer()``, it will be automatically
+When you are using an inline script, after you instantiate the object like ``tracer = VizTracer()``, it will automatically be
```

Comment:
Change this to:

When you instantiate the `VizTracer` object like ``tracer = VizTracer()`` in your script, XX

Also it will be automatically registered and it will automatically be registered are both correct I believe, and I like it when automatically is right next to the verb. :)